pyCGM_Single/pycgm_embed.py

```diff
@@ -41,6 +41,16 @@
     import pyCGM_Single.pycgmCalc as pycgmCalc
 
 def getfilenames():
+    """Gets the filenames of the dynamic trial, static trial, vsk file, and
+    output file and returns their file paths. The example filenames below
+    are found in the SampleData folder in the git repository.
+
+    Returns
+    -------
+    tuple
+        Returns a tuple of four strings that includes the paths to
```

Comment:
I am not sure the sentence should start with returns, rather should describe the data that is being returned